node/engine/chainservice/simulated_backend_chainservice_test.go

```diff
@@ -65,21 +60,52 @@ func TestSimulatedBackendChainService(t *testing.T) {
 		},
 		ChannelNonce:      37140676580,
 		AppDefinition:     bindings.ConsensusApp.Address,
-		ChallengeDuration: 0,
+		ChallengeDuration: 1000,
```

Comment:
Nit but you could throw this in a const and maybe add a comment about how long this is (1000 seconds right?)

Reply:
[9a9f020](https://github.com/statechannels/go-nitro/pull/1619/commits/9a9f020bf9f4aa55d2c6ca5bbe0250eb8d03edf9)